src/modules/nav/NavContainer.jsx

```diff
@@ -24,7 +24,17 @@ const LoggedInLinks = () => (
 );
 
 export default function Nav({user}) {
-	return <StyledNav className="nav">{user ? <LoggedInLinks /> : <LoggedOutLinks />}</StyledNav>;
+	var page = {};
+	if (location.pathname == '/') {
+		page.landing = ' transparent';
+	} else {
+		page.landing = '';
+	}
+	return (
+		<StyledNav className={'nav' + page.landing}>
+			{user ? <LoggedInLinks /> : <LoggedOutLinks />}
+		</StyledNav>
```

Comment:
Back to your old habits mang.

This whole thing could be:
```jsx
export default function Nav() {
  return (
    <StyledNav transparent={location.pathname === ROUTES.LANDING} className="nav">
      <Link to={ROUTES.LANDING}>
        OPENRECORD
      </Link>
    </StyledNav>
  );
}

const StyledNav = styled.nav`
  background: ${props => (props.transparent ? 'transparent' : 'rgb(25, 25, 25)')};
  ...
`
```

Reply:
😬it's really due to my lack of knowledge of the appropriate syntax. I just made the fix. Thanks for spelling it out!